datalad/cmd.py

```diff
@@ -692,3 +693,151 @@ def run(self, cmd, env=None, *args, **kwargs):
         # All communication here will be returned as unicode
         # TODO: do that instead within the super's run!
         return assure_unicode(out), assure_unicode(err)
+
+
+def readline_rstripped(stdout):
+    #return iter(stdout.readline, b'').next().rstrip()
+    return stdout.readline().rstrip()
+
+
+@auto_repr
+class BatchedCommand(object):
+    """Container for an annex process which would allow for persistent communication
```

Comment:
This copied over code has some references to "annex".  I'll push a clean-up commit after the current test run completes.

Reply:
Thx.